docs/UserGuide.md

```diff
@@ -72,9 +72,9 @@ Shows a message listing out all the available commands and their purpose.
 
 Format: `help`
 
-For more information regarding the comamnd formats and examples, press `F1` to open up a help window (as shown in the picture below).
+For more information regarding the command formats and examples, press `F1` to open up a help window (as shown in the picture below).
 To close the help window, you can simply press `esc` on your keyboard (other methods such as `alt`+`F4` and clicking on
-the red `X` button on the top right corner of the window also works).
+the red `X` button in the top right corner of the window also works).
```

Comment:
Not issue with this pull request, but in the meantime can you clarify that these methods (alt+F4 and red X) **only apply for Windows**

Reply:
Can check this and the below formatting as well